Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -55,42 +101,73 @@ def sim_time_parameters(file_path):
     Must be written as dt = X.Y in the module file
     
     @param  file_path   str of file name containing parameters to be modified
+    @return dt          Simulation time step specified in CellModeller module file
```

Comment:
As I wrote in the previous comments, you don't need to find the 'dt' value by searching in the simulation file, this variable can be defined in the main function as a global variable and let a user define it.